src/commands/runDraftTool/runDraftSetupGH.ts

```diff
@@ -32,46 +40,92 @@ async function multiStepInput(context: ExtensionContext, destination: string) {
         ghRepo: string;
 	}
 
+    const defaultAppName = basename(destination);
+
+	const filteredRepositories = git.repositories.filter(r => r.rootUri.fsPath === destination);
+	const firstFiltered = filteredRepositories[0];
+	const firstRemote = firstFiltered && firstFiltered.state.remotes[0];
+	const firstPushUrl = firstRemote && firstRemote.pushUrl;
+	const defaultRepo = firstPushUrl || 'https://github.com/username/repo';
```

Comment:
💡 Idea share/wondering - Cool, what does this default arbetrary gihub names leads to what scenario? `https://github.com/username/repo` Also the above var names with `first...` what do they signify?